watson_developer_cloud/tone_analyzer_v3.py

```diff
@@ -13,15 +13,25 @@
 # limitations under the License.
 
 """
-The v3 Tone Analyzer service
-(https://www.ibm.com/watson/developercloud/tone-analyzer.html)
+The IBM Watson™ Tone Analyzer Service uses linguistic analysis to detect
```

Comment:
Good point! We wouldn't normally include the trademark symbol if writing the docstring by hand, so I updated the generator to remove it.